modules/ROOT/pages/_partials/connection-node-warning.adoc

```diff
@@ -0,0 +1 @@
+IMPORTANT: The connection must point to a node running the data service.
```

Comment:
Is this an admonition that will see frequent use in the docs? I'm not sure the abstraction is required for two instances

Reply:
Probably not, actually; I think I'm only using it in two places. I tend to use includes when the text is going to appear in two or more places.

It's a habit from my programming days, I think. I can certainly bin it though.

Reply:
No, hang on; just remembered why I labelled the branch with a version number. I'm going to use the same warning in the other versions.

I think I'll keep it as is.

Reply:
Nope. I've changed my mind. It's overkill for one  single line of text, even if's going in multiple versions.